suites/tentacle/nvmeof/tier-3_nvmeof_librbd_vs_nvme_128kbs_10vols_bm.yaml

```diff
@@ -0,0 +1,177 @@
+# Basic IO Perf comparison test LibRBD vs NVMeoF protocols
+# cluster configuration: conf/squid/nvmeof/ceph_nvmeof_sanity.yaml
```

Comment:
@HaruChebrolu Please remove all librbd vs nvmeof tests

Reply:
Does not make sense as we are not testing it

Reply:
I feel, we can have it in cephci irrespective of usage. We can removal its usage in metadata. WDYT?